lxd/device/gpu_physical.go

```diff
@@ -98,10 +101,192 @@ func (d *gpuPhysical) Start() (*deviceConfig.RunConfig, error) {
 	return d.startContainer()
 }
 
+// startCDIDevices starts all the devices given in a CDI specification:
+// * `unix-char` (representing the card and non-card devices)
+// * `disk` (representing the mounts)).
+func (d *gpuPhysical) startCDIDevices(configDevices *cdi.ConfigDevices, hooks *cdi.Hooks, runConf *deviceConfig.RunConfig) error {
```

Comment:
only runConf is modified right? So do the other args need to be passed as pointers?

Reply:
yes, we can change that. Fixed.